src/http/message.h

```diff
@@ -0,0 +1,39 @@
+#ifndef MESSAGE_H
+#define MESSAGE_H
+
+#include <string>
+#include <vector>
+
+/*
+ * Interface for HTTP messages. Used to hold common code
+ * for HTTP Requests and Responses
+ */
+class Message {
+protected:
+	// Effectively namespaced constant for carriage returns
+
+	std::vector<std::string> headers_;
```

Comment:
I'm following the naming convention [https://google.github.io/styleguide/cppguide.html#Variable_Names](here) cause I saw it in lecture lol. I'm not attached to anything but I would prefer consistency.